### PR TITLE
feat(excersises): excercises functions implementstion

### PR DESCRIPTION
**Pull request by shevqnov**

### CODE REVIEW

File: Exercises/1-callback.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
 'use strict';
 
-const iterate = (obj, callback) => null;
+const iterate = (obj, cb) => {
+  Object.keys(obj).forEach(key => {
```
Comment (tshemsedinov) [inline Exercises/1-callback.js]: Use loops, forEach is not a good idea here.

File: Exercises/3-wrapper.js [hunk 2]
```diff
@@ -1,5 +1,25 @@
 'use strict';
 
-const contract = (fn, ...types) => null;
+const contract = (cb, ...types) => (...args) => {
```
Comment (tshemsedinov) [inline Exercises/3-wrapper.js]:
```suggestion
const contract = (fn, ...types) => (...args) => {
```

File: Exercises/2-closure.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 'use strict';
 
-const store = x => null;
+const store = value => () => value;
```
Comment (tshemsedinov) [inline Exercises/2-closure.js]:
```suggestion
const store = x => () => x;
```

File: Exercises/3-wrapper.js [hunk 4]
```diff
@@ -1,5 +1,25 @@
...
+  args.forEach((arg, idx) => {
```
Comment (tshemsedinov) [inline Exercises/3-wrapper.js]: Use loops

**Comment by tshemsedinov** [comment]
`exercises`, not `exersises`

File: Exercises/1-callback.js (hunk 1 above)
Comment (shevqnov) [reply Exercises/1-callback.js]: ✅

File: Exercises/3-wrapper.js (hunk 2 above)
Comment (shevqnov) [reply Exercises/3-wrapper.js]: done

File: Exercises/2-closure.js (hunk 3 above)
Comment (shevqnov) [reply Exercises/2-closure.js]: ✅

File: Exercises/3-wrapper.js (hunk 4 above)
Comment (shevqnov) [reply Exercises/3-wrapper.js]: ✅

**Pull request closed without merging**